src/npyffi/mod.rs

```diff
@@ -10,13 +10,18 @@
 )]
 
 use std::mem::forget;
-use std::os::raw::c_void;
+use std::os::raw::{c_uint, c_void};
 
 use pyo3::{
+    sync::GILOnceCell,
     types::{PyAnyMethods, PyCapsule, PyCapsuleMethods, PyModule},
     PyResult, Python,
 };
 
+pub const API_VERSION_2_0: c_uint = 0x00000012;
+
+pub static API_VERSION: GILOnceCell<c_uint> = GILOnceCell::new();
```

Comment:
I think we should not make the direct cache public:

```suggestion
static API_VERSION: GILOnceCell<c_uint> = GILOnceCell::new();
```

Reply:
Done.